cpp/include/IceSSL/Plugin.h

```diff
@@ -31,6 +31,62 @@
 namespace IceSSL
 {
 
+/**
+ * The reason for an IceSSL certificate verification failure.
+ */
+#ifdef ICE_CPP11_MAPPING
+enum class TrustError : unsigned char
+#else
+enum TrustError
+#endif
+{
+    /** The certification verification succeed  */
+    NoError = 0,
+    /** The certificate chain length is greater than the specified maximum depth **/
+    ChainTooLong,
+    /** The certificate is explicitly distrusted **/
+    ExplicitDistrust,
+    /** The X509 chain is invalid because a certificate has excluded a name constraint **/
+    HasExcludedNameConstraint,
+    /** The certificate has an undefined name constraint"**/
+    HasNotDefinedNameConstraint,
```

Comment:
HasNonDefinedNameConstraint? The comment has an extra " as well.

Reply:
Fixed